### PR TITLE
Added deserializing JSON from CMG

### PR DESCRIPTION
**Pull request by ptgullas**
Hey, I added a CmgJsonConverter class to handle deserializing the Json, a few Dto classes to match what's in the Json, as well as an XUnit test project. We'll probably have to ignore some of the changes in the .sln file (I'm using VS2019), and these .nuget.dgspec.json files.

### CODE REVIEW

**Comment by ptgullas** [comment]
I guess I should've made a branch for this.

**Comment by ksigWyatt** [comment]
Nice addition of the test classes and stuff too.

File: API/CMGScripturesAPI/CMGScripturesAPI.Repos.Tests/CMGScripturesAPI.Repos.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```
Comment (ksigWyatt) [inline API/CMGScripturesAPI/CMGScripturesAPI.Repos.Tests/CMGScripturesAPI.Repos.Tests.csproj]: Would you change this to target `netcoreapp2.2`? We need to be able to use just one runtime for everything.

**Comment by ksigWyatt** [comment]
Thank you very much!

**Pull request merged**